### DISCUSSION TITLE
Cluster becomes inaccessible in ~24 hours after creation :)

### DISCUSSION THREAD

**Discussion by madbbb** [syself/cluster-api-provider-hetzner#688] [Q&A] [answered]
I know it sounds ridiculous, but this is the third cluster in row that I lost. It all started a month ago - my first  cluster was working fine for 3 months and then I noticed that API LoadBalancer is red - it can't reach any of 3 control planes. I checked servers tab and see all the servers are here and green, but I can't ping or ssh to any of them and all the servers are freshly recreated. I tried to login with recovery console to one of the oldest control planes and I see that it has no access to the internet or private network at all.  The only access to the nodes I have is by recovery console and its resolution is so small so every simple command output is very painful to read. Pease advise what logs to check, what could be the problem?

**Comment by madbbb** [comment]
I think it is related to DHCP leases updated in 24 hours and I noticed that ipv6 network is working fine, only ipv4 not working at all.

**Comment by madbbb** [comment]
Found the same issue https://github.com/kubermatic/machine-controller/issues/1587

**Comment by batistein** [comment, marked as answer]
This is solved since: https://github.com/syself/cluster-api-provider-hetzner/pull/687

**Comment by madbbb** [comment]
@batistein thank you for explanation and for your great work I really love it.
